js/build.js

```diff
@@ -9,6 +9,16 @@
   // ============================
 
   var collapsibleTemplate = Fliplet.Widget.Templates['templates.collapsible'];
+
+  function resizeWindow() {
+    if (Modernizr.windows) {
```

Comment:
@hcarneiro [This solution](https://github.com/Fliplet/fliplet-widget-accordion-start/pull/4#issuecomment-396308614) looks like it directly detects JavaScript features instead of checking for browsers/OS. Have you tried it?